### ISSUE TITLE
feature request : send event to obs-browser

### ISSUE THREAD

**Issue by Grummfy** [faithoflifedev/obs_websocket#25]
Hello,
I think a nice feature using the _CallVendorRequest_ could be a specific call for the browser.

Here is an example :

```dart
    var response = await obsWebSocket.send('CallVendorRequest', {
      'vendorName': 'obs-browser',
      'requestType': 'emit_event',
      'requestData': {
        'event_name': 'obs-websocket-test-event',
        'event_data': {'my':'data', 'that':'will be displayed in an event'},
      },
    });

    print(response.toString());
    print(response?.requestType);
    print(response?.requestId);
    print(response?.requestStatus);
    print(response?.responseData);
```

this will display as 

```
flutter: {"requestType":"CallVendorRequest","requestId":"e2bd885e-a382-4090-964b-98f90a76e7f9","requestStatus":{"result":true,"code":100,"comment":null},"responseData":{"requestType":"emit_event","responseData":{},"vendorName":"obs-browser"}}
flutter: CallVendorRequest
flutter: e2bd885e-a382-4090-964b-98f90a76e7f9
flutter: {"result":true,"code":100,"comment":null}
flutter: {requestType: emit_event, responseData: {}, vendorName: obs-browser}
```

In the browser, we can see the result

```js

window.addEventListener('obs-websocket-test-event', function(event) {
	console.log(event.detail) //  {"my":"data","that":"will be displayed in an event"}
})

```

I think that adding an event to the browser could be really interesting, because it is a plugin that is really use everywhere.

https://github.com/obsproject/obs-browser/pull/340 for the browser integration

**Comment by faithoflifedev** [comment]
Hi @Grummfy,

Are you recommending something like:

`await obsWebSocket.browserEvent(eventName: 'obs-websocket-test-event', eventData: {'my': 'data', 'that': 'will be displayed in an event'});`

**Comment by Grummfy** [comment]
yes, something like this.

**Comment by faithoflifedev** [comment]
The package released today (v5.0.0+9) allows:

`await obsWebSocket.general.obsBrowserEvent(
  eventName: 'obs-websocket-test-event',
  eventData: {
    'my': 'data',
    'that': 'will be displayed in an event',
  },
);`

**Comment by Grummfy** [comment]
thanks a lot
